ivy/functional/ivy/elementwise.py

```diff
@@ -1530,6 +1530,68 @@ def expm1(
         an array containing the evaluated result for each element in ``x``. The returned
         array must have a floating-point data type determined by :ref:`type-promotion`.
 
+    This function conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.elementwise_functions.add.html>`_ # noqa
```

Comment:
Please update the link on line 1535 to the correct one for `expm1`

Reply:
Done.